RafRate.API/Controllers/AuthenticationController.cs

```diff
@@ -8,10 +11,32 @@ namespace RafRate.API.Controllers
     public class AuthenticationController : ControllerBase
     {
         private readonly IAuthenticationService _authenticationService;
-
-        public AuthenticationController(IAuthenticationService authenticateService)
+        private readonly AppDbContext _dbContext;
+        public AuthenticationController(IAuthenticationService authenticateService, AppDbContext dbContext)
         {
             _authenticationService = authenticateService;
+            _dbContext = dbContext;
+        }
+        
+        // test method
+        [HttpGet]
+        [Route("Users")]
+        public async Task<IActionResult> GetUsers()
+        {
+            List<UserEntity> listUsers = _dbContext.Users.ToList();
+            try
```

Comment:
Why do we need try here?

Reply:
It's just test method, will be deleted in the future